plugins/ti-tps6598x/fu-ti-tps6598x-pd-device.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+ OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-ti-tps6598x-common.h"
+#include "fu-ti-tps6598x-device.h"
+#include "fu-ti-tps6598x-firmware.h"
+#include "fu-ti-tps6598x-pd-device.h"
+
+struct _FuTiTps6598xPdDevice {
+	FuDevice parent_instance;
+	guint8 target;
+};
+
+G_DEFINE_TYPE(FuTiTps6598xPdDevice, fu_ti_tps6598x_pd_device, FU_TYPE_DEVICE)
+
+// FIXME move to common code
```

Comment:
missed FIXME?

Reply:
Done in https://github.com/fwupd/fwupd/pull/5327 -- will remove on rebase.